tests/test_field_validation.py

```diff
@@ -0,0 +1,76 @@
+"""
+Server responds with a 400 response when invalid information is provided in the body of the request
+
+NHS Number must be 10 digits long
+DOB must be in the format of YYYY-MM-DD
+"""
+import requests
+import pytest
+
+
+NHS_NUMBER = ["012345678", "01234567890", "abcdefghij"]
+DOB = ["1990-10-1", "1990-1-10", "90-10-10", "10-12-1990", "1-MAY-2000", "1990/01/01"]
+

```

Comment:
You could expand on this and add tests for the uuid validation on:

* routingPlanId
* messageBatchReference
* messageReference

Reply:
Good shout! I'll add those in

Reply:
the new tests look good, all passing on the CI, resolving.